packages/vscode-extension/package.json

```diff
@@ -982,6 +982,12 @@
               "yo": true,
               "sharepointYeomanGenerator": true
             }
+          },
+          "fx-extension.SPFxVersion":{
+            "type": "string",
+            "description": "Select used SPFx version. (Requires reload of VS Code)",
```

Comment:
Select what version of SPFx to use. (Requires reload of VS Code)

Reply:
Updated